### DISCUSSION TITLE
How to hide Thumbs in V5?

### DISCUSSION THREAD

**Discussion by wpdevnettr** [fancyapps/ui#445] [Q&A]
Greetings, I am V4 user and I was able to hide Thumbs images easily.

But I couldn't see this feature in v5.

Here is my V4 code, but it didn't work in V5.

 Fancybox.bind('[data-fancybox="gallery"]', {
  Thumbs: false,
});

**Comment by fancyapps** [comment]
Hi,

That works the same, demo - https://jsfiddle.net/za59emb4/

**Reply by fancyapps** [reply]
But, if you just want to hide initially, it would be like this:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  Thumbs : {
    showOnStart: false
  }
});      
```

https://jsfiddle.net/za59emb4/1/

**Reply by wpdevnettr** [reply]
Thank You.

**Reply by putimir** [reply]
Can you hide thumbs for a specific slide or slide type (eg. IFRAME)? Thank you!

**Reply by fancyapps** [reply]
Well, it depends on your project, but typically you just set different options per group, like this - https://jsfiddle.net/zgn1wvbL/
